ui/app/services/auth.js

```diff
@@ -441,4 +441,22 @@ export default Service.extend({
       backend: BACKENDS.findBy('type', backend),
     });
   }),
+
+  getOktaNumberChallengeAnswer(nonce) {
+    let namespace = 'undefined';
+    const url = `/v1/auth/okta/verify/${nonce}`;
+    return this.ajax(url, 'GET', { namespace }).then(
+      (resp) => {
+        return resp.data.correct_answer;
```

Comment:
I talked to Calvin about the 404 error I was getting and he told me that we get a 404 if the user did not verify "Yes it's me" on their phone and Okta has not sent the number challenge yet. Once the user clicks "Yes it's me", it'll give the proper challenge number response and the `correct_answer` would be given. From testing the polling I never saw a 204 response since the error is always 404 until the user verifies it is them. Should I check if the value is 204 just in case as well?

Reply:
That's great context, thank you. I would double check with Calvin that the endpoint does return a `204` response after the user responds `yes it's me` and if the okta response to Vault backend is slow. In other words, check if it's *possible* for the endpoint to respond with a `204` and then we can check in the response here if it is possible.

Reply:
Confirmed with Calvin that the only responses for this endpoint are 202 and 404 since `verify/<nonce>`  is a read operation, and if we return a nil response and no error, it gets translated to a 404.